<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDocumentDialog</class>
 <widget class="QDialog" name="NewDocumentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Document</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>301</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>471</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Image Parameters</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>29</y>
      <width>451</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="SDF::UILayer::Gui::View::Qt::CustomWidgets::LengthQuantityEdit" name="widthSelector">
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="SDF::UILayer::Gui::View::Qt::CustomWidgets::LengthQuantityEdit" name="heightSelector">
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Resolution: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="SDF::UILayer::Gui::View::Qt::CustomWidgets::ResolutionQuantityEdit" name="resolutionSelector">
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Color Model:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="colorModelSelector">
         <property name="toolTip">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>at</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="bitDepthSelector"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>bits/channel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Initial Background:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="initialBackgroundSelector"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Preset:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="presetSelector">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="manageButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Manage...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Memory req for one layer:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="memoryReqIndicator">
      <property name="text">
       <string>0.00 MiB</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SDF::UILayer::Gui::View::Qt::CustomWidgets::LengthQuantityEdit</class>
   <extends>QWidget</extends>
   <header>SDF/UILayer/Gui/View/Qt/CustomWidgets/LengthQuantityEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>SDF::UILayer::Gui::View::Qt::CustomWidgets::ResolutionQuantityEdit</class>
   <extends>QWidget</extends>
   <header>SDF/UILayer/Gui/View/Qt/CustomWidgets/ResolutionQuantityEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewDocumentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
